.gitignore

```diff
@@ -13,3 +13,4 @@ TAGS
 /scripts/sources-*/
 /scripts/sources.local.conf
 /scripts/rest_examples/*.local.http
+/dao_tests.test
```

Comment:
I have refactored the Makefile so this is not necessary and dropped this in: https://github.com/RHEnVision/provisioning-backend/pull/135

You can drop this from here.

Reply:
Or I can drop it from my PR so this can be done in a separate PR if you like.

Reply:
> Or I can drop it from my PR so this can be done in a separate PR if you like.

No problem, I will just drop it from this PR, leave it in yours.